plaso/cli/log2timeline_tool.py

```diff
@@ -401,7 +400,10 @@ def ExtractEventsFromSources(self):
     if single_process_mode:
       extraction_engine = single_process_engine.SingleProcessEngine()
     else:
-      extraction_engine = multi_process_engine.TaskMultiProcessEngine()
+      extraction_engine = multi_process_engine.TaskMultiProcessEngine(
```

Comment:
Rather than adding args to the engine constructor, please add the worker_* values to the [processing_configuration](https://github.com/log2timeline/plaso/blob/35082c09e525fac88daa2b522d9d546f7d051184/plaso/engine/configurations.py#L172) object that gets passed to ProcessSources().

This is where all the other processing-specific args are, with apparently a couple of exceptions that should also move. Having two different paths for configuring an engine to process something adds extra complexity, and I don't see any advantage for having these different paths.

Reply:
The reasoning behind moving this to the constructor was that these values don't change once set and are engine specific.